frontend/components/panels/DesktopClassPanel.css

```diff
@@ -73,33 +70,9 @@
     text-align: center !important;
```

Comment:
Important is effectively an anti-pattern in CSS; we should find a way to avoid this.

Reply:
Yeah....  Most (all?) of the CSS with the `!important` tags are overriding the CSS built into the UI library. Can't really do much about this unless we want to copy a bunch of CSS out of the UI library... :/